meta.go

```diff
@@ -241,16 +242,110 @@ func parseStringList(list interface{}) []string {
 	return result
 }
 
-var termNameRE = regexp.MustCompile("^[a-z]+([a-z0-9-]+)(/[0-9]+)?$")
+var validTermName = regexp.MustCompile(`^[a-zA-Z][\w-]+$`)
 
-func checkTerm(s string) error {
-	match := termNameRE.FindStringSubmatch(s)
-	if match == nil {
-		return fmt.Errorf("invalid term name %q: must match %s", s, termNameRE.String())
+// TermID represents a single term id. The term can either be owned
```

Comment:
As discussed online, I still think that "terms" is better to use throughout. "term" isn't a thing and has other connotations in a computery context.